packages/lib/timeFormat.ts

```diff
@@ -8,6 +8,11 @@ import { localStorage } from "@calcom/lib/webstorage";
 
 const is24hLocalstorageKey = "timeOption.is24hClock";
 
+export enum TimeFormat {
```

Comment:
I found `h:mma` etc very confusing throughout the code, so I converted in to an enum for way better readability.